generators/server/templates/pom.xml.ejs

```diff
@@ -365,8 +370,10 @@
             <artifactId>spring-boot-starter-data-neo4j</artifactId>
         </dependency>
         <dependency>
-            <groupId>eu.michael-simons.neo4j</groupId>
-            <artifactId>neo4j-migrations-spring-boot-starter</artifactId>
+            <groupId>org.liquibase.ext</groupId>
+            <artifactId>liquibase-neo4j</artifactId>
+            <!-- Inherited version from Spring Boot can't be used because of regressions -->
+            <version>${liquibase.version}</version>
```

Comment:
note: update to 4.8.0 would be better
JHipster currently depends on Liquibase core 4.6.1 which is the first ever release of the Neo4j plugin... There have been security fixes and improvements since.

Reply:
Spring-boot 2.6.3 is not compliant with liquibase 4.6.1+, only 2.7.0 that is not yet published